### ISSUE TITLE
TF 1.0 support

### ISSUE THREAD

**Issue by bkovalev** [CGCL-codes/Tensorflow-RDMA#1]
Hi,
Can I run it on TF 1.0 and CUDA 8 on Ubuntu 16.04?

I can't see this path src/core/ in the github.

When I can find changed files?

**Comment by xuanhuashi** [comment]
I don't think it works on TF 1.0, but you can modify tf 1.0 according to our modifications.

**Comment by lwdhw1987** [comment]
after I  invoke  git clone  https://github.com/CGCL-codes/Tensorflow-RDMA， and download the code,  I can't find the modified  files in this github code. for example , src/core/ext/transport/chttp2/client/insecure/channel_create.c is missing.

**Comment by kaixili** [comment]
Hi lwdhw1987, bkovalev,
Distributed communication of tensorflow is in git submodule grpc. All modified file is in grpc.
Use `$git clone src/grpc.git ` get modified  grpc module.

**Comment by andrewlixin** [comment]
In config of src/grpc.git, the remote origin is your local folder:

[remote "origin"]
	url = /home/uniquesc/likaixi/grpc
